src/components/document/TableView/TableView.jsx

```diff
@@ -1,6 +1,78 @@
-const TableView = () => { 
-    return <div>
-        TableView
+import CheckIcon from "@/assets/check_fill.svg";
+import CloseIcon from "@/assets/close_fill.svg";
+import CommentIcon from "@/assets/comment_fill.svg";
+import NavigateToIcon from "@/assets/navigate_to_icon.svg";
+import { Slider } from 'antd';
+import { useState } from "react";
+
+const RedactionFilterDropdownMenu = () => {
+    //TODO: Some sort of dropdown
+    return <div className="w-full text-end">
+        Accepted Suggestions 
```

Comment:
Might want to add some padding/bolding to make this look more like a title/label. But I suppose we'll probably do that once we have the dropdown anyways so not super important